pkgs/development/tools/misc/dlang-dfmt/default.nix

```diff
@@ -0,0 +1,2 @@
+{ pkgs ? import <nixpkgs> { } }: pkgs.callPackage ./dfmt.nix { }
```

Comment:
I think this file doesn't belong here. The file currently named `dfmt.nix` should be renamed to `default.nix`, and this can be deleted.